DistributionPackages/Neos.NeosIo/Configuration/NodeTypes.MultiColumn.yaml

```diff
@@ -43,22 +43,14 @@
                 label: '33% / 66%'
   childNodes:
     column0:
+      type: 'Neos.NeosIo:Content.Column'
```

Comment:
I assume we should have a node migration for this, or how will the behaviour be?

Reply:
Wonder because also old elements I did not touch worked layout wise.
The reason is that the styling is now parent and not item oriented.

So it is more or less for elements that need the buttons.
But can try the migration thing.

When you have no link set nothing changes normally .

Reply:
I would assume that they would complain now when doing a `node:repair` at in the worst case if someone doesn't take care loose their columns.